ttnn/cpp/ttnn/operations/eltwise/unary_backward/unary_backward.hpp

```diff
@@ -47,7 +47,20 @@ struct ExecuteUnaryBackward {
         return op_type(grad_tensor_arg, input_tensor_arg, alpha, output_memory_config);
         }
 
-    //Type 2: Type 1 with 2 float
+    //Type 2: Type 1 with 1 int
+
+    static std::vector<ttnn::Tensor> execute_on_worker_thread(
+        const Tensor &grad_tensor_arg,
+        const Tensor &input_tensor_arg,
+        int n,
+        const std::optional<MemoryConfig> &memory_config = std::nullopt) {
+
+        auto op_type = UnaryBackwardFunction::get_function_type1_w_int(unary_backward_op_type);
```

Comment:
What's the plan here?  Are we not going to take advantage of the fact that unary_backward_op_type is a template argument and this function could be provided more performantly?   https://tenstorrent.slack.com/archives/C05GH08C58F/p1720482862057979